andela/trime
# Improve E2E setup

#### What does this PR do?
Improves the E2E tests setup so E2E tests can be run in a single command.

#### Description of Task to be completed?
Refactor the application configuration in order to have a test environment different from the development environment. Configure E2E test setup so that E2E tests can be run with a single command.

#### How should this be manually tested?
Clone this repository.
Install the dependecies.
Run `npm test` to verify that another environment is used.
Run `npm run test:e2e` to run the end to end tests.

#### What are the relevant pivotal tracker stories?
[Improve E2E setup](https://www.pivotaltracker.com/projects/1454736/stories/118455043)
